kokoro/checks/license_headers/check.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Fail on any error.
+readonly DIR="/mnt/github/orbitprofiler"
+readonly SCRIPT="${DIR}/kokoro/checks/license_headers/check.sh"
+
+if [ "$0" == "$SCRIPT" ]; then
+  # We are inside the docker container
+
+  cd "$DIR"
+  REFERENCE="origin/master"
+  MERGE_BASE="$(git merge-base $REFERENCE HEAD)" # Merge base is the commit on master this PR was branched from.
+  LICENSE_HEADER_MISSED=""
+
+  echo -e "The following files are missing a license/copyright header:\n"
+  while read file; do
+    if [ ! -f "$file" ]; then
+      continue
+    fi
+
+    if ! grep 'Copyright (c) 2020 The Orbit Authors. All rights reserved.' "$file" > /dev/null; then
```

Comment:
This will break when we add files in a few month - would it make sense to replace "2020" with a "\d{4}" regex?

Reply:
Yes, definitely. Thanks!

Reply:
Done. PTAL!

Reply:
LGTM, thanks!